posts/how-kubernetes-add-to-your-business.md

```diff
@@ -0,0 +1,60 @@
+---
+authors:
+- Jeet Gajjar
+tags:
+- kubernetes
+- docker
+- microservices
+
+date: 
+title: "How Kubernetes Adds to Your Business"
+image: https://github.com/ippontech/kakorrhaphio/blob/master/images/2019/6/kubernetes-image.png
+---
+
+Monolithic applications have been the industry norm for years, however, we here at Ippon have noticed a shift in the 
+industry wherein many companies in the insurance to e-commerce to banking industries are bound for generating more 
+focused and specific applications. These applications exist in the form of [microservices](https://blog.ippon.tech/monolithic-to-microservice-consistency-in-distributed-systems/).  When talking about microservices, its difficult _not_ to talk about containers and container orchestration tools.
+When reading into microservices, you and your developers will ask yourselves: `Would you want to spin up a VM,  
+install an OS, find dependencies, manage security, etc or would you like to spin up a container which does all that 
+for you in seconds?`
+
+Today, we're going to discuss what is Kubernetes and why your business would benefit from this technology. First
+we'll briefly discuss the differences in microservices and monoliths.
+
+### A Brief History in Microservices
+Before microservices, we had (and still have) monoliths. Applying good development practices, its best to break up 
+your main component into modular pieces. These modular pieces, in the form of microservices, are transforming into 
+containers.
+
+Making monoliths modular into microservices have real cost saving benefits both financially and in developer time:
```

Comment:
*Making monoliths modular into microservices are a dual threat of sorts when it comes to savings -- both in terms of financial costs and also in development time.